xapian-letor/api/featuremanager_internal.cc

```diff
@@ -125,6 +125,9 @@ FeatureManager::Internal::load_relevance(const std::string & qrel_file)
 
     string inLine;
     ifstream myfile (qrel_file.c_str(), ifstream::in);
+    if(!myfile.good()){
+        cout << "No Qrel file found" << endl;
```

Comment:
This should probably raise an exception instead. It certainly can't proceed, can it?